File: PostApp/src/app/Models/post.model.ts


export class Post {
    id: string;
    authorId: string;
    title: string;
    content: string;
    status: string | null;
    creationDate: string;
    approvalDate: string|null;


    constructor() {
        this.id = "";
        this.authorId = "";
        this.title = "";
        this.content = "";
        this.status = "false";
        this.creationDate = "";
        this.approvalDate = null
    }
}

File: TESTAPI1.Application/Models/PostModel.cs
namespace TESTAPI1.Application.Models
{
    public class PostModel
    {
        public Guid Id { get; set; }
        public string Title { get; set; }
        public string Content { get; set; }
        public Guid AuthorId { get; set; }
        public DateTime CreationDate { get; set; } 
        public DateTime? ApprovalDate { get; set; }
        public bool? Status { get; set; }
    }
}
